client/src/components/SubmitCode.js

```diff
@@ -1,5 +1,5 @@
 import React, { PropTypes, Component } from 'react'
-import { connect } from 'react-redux'
+import { connect } from 'react-redux';
```

Comment:
I've been doing them, but i think we should actually just leave out the semicolons.